enlighten/timing/BatchCollection.py

```diff
@@ -499,6 +499,11 @@ def stop(self):
         self.running = False
         self.timer_measurement.stop()
         self.timer_batch.stop()
+
+        if self.laser_mode != "manual":
+            log.debug(f"BatchCollection.stop: disabling all lasers (laser_mode {self.laser_mode})")
+            self.ctl.laser_control.set_laser_enable(False, all_=True)
+
```

Comment:
This fixed the issue where people were hitting "Stop" (⏹️) during a BatchCollection, and it wasn't turning off the laser, even if the laser had been automatically enabled as part of the BatchCollection settings.